src/compiler/sys/stencil-sys.ts

```diff
@@ -506,6 +505,8 @@ export const createSystem = (c?: { logger?: Logger }) => {
     return results;
   };
 
+  const fetch = global.fetch;
```

Comment:
For my own knowledge, should we be assigning `fetch` to `global` in our new `fetch` method [here](https://github.com/ionic-team/stencil/pull/2964/commits/d52a7d1134d70fe153dfa9840c313cfc7210eccb#diff-585e96e5364c622963a06a9d71c01a77ad8816bc84e97763dcefe7920176dd50R102)?

Reply:
Yes, and that has been updated!